app/controllers/sufia/admin/admin_sets_controller.rb

```diff
@@ -30,6 +30,7 @@ def index
       add_breadcrumb t(:'sufia.toolbar.admin.menu'), sufia.admin_path
       add_breadcrumb t(:'sufia.admin.sidebar.admin_sets'), sufia.admin_admin_sets_path
       @admin_sets = CurationConcerns::AdminSetService.new(self).search_results(:read)
+      @presenter = presenter_class.new(self, current_ability)
```

Comment:
The first argument to AdminSetPresenter#initialize is a SolrDocument (see https://github.com/projecthydra/curation_concerns/blob/master/app/presenters/curation_concerns/collection_presenter.rb#L11).  You've passed a instance of the controller.